cmd/snapctl/main.go

```diff
@@ -48,7 +50,9 @@ func main() {
 	app.Commands = append(commands, tribeCommands...)
 	sort.Sort(ByCommand(app.Commands))
 	app.Before = beforeAction
-	app.Run(os.Args)
+	if app.Run(os.Args) != nil {
```

Comment:
This would be the place to print the errors passed back instead of with `fmt.Println`